prboom2/CMakeLists.txt

```diff
@@ -183,5 +183,20 @@ set(PRBOOM_OUTPUT_PATH ${CMAKE_BINARY_DIR})
 set(WAD_DATA prboom-plus.wad)
 set(WAD_DATA_PATH "${PRBOOM_OUTPUT_PATH}/${WAD_DATA}")
 
+if( WIN32 )
+	set( DOCDIR doc CACHE STRING "Directory to install documentation into" )
+else()
+	set( DOCDIR "${CMAKE_INSTALL_PREFIX}/share/doc/${PROJECT_NAME}" CACHE STRING "Directory to install documentation into" )
+endif()
+install(DIRECTORY doc/ DESTINATION ${DOCDIR} COMPONENT "Documentation")
+
+if( WIN32 )
+	set( MANDIR man CACHE STRING "Directory to install manpages into" )
+else()
+	set( MANDIR "${CMAKE_INSTALL_PREFIX}/share/man" CACHE STRING "Directory to install manpages into" )
+endif()
+install(DIRECTORY man/man5/ DESTINATION "${MANDIR}/man5" COMPONENT "Man5 manpages")
```

Comment:
Couldn't we just `install(FILES doc/*.5 ...` or similar here? Then we souldn't have to move the files around at all.

Reply:
I don't believe wildcards are allowed when using `install(FILES ...`; I could list each file out explicitly. However, this does present the problem that the manpages would be installed in two locations because they will also be picked up by the earlier `install(DIRECTORY doc/ ...` line.

Reply:
Do you think I should list out all the documentation files explicitly -- I think this is probably the style CMake is expecting in fact, and it'll prevent the double installation.